src/Common/src/Common.Expression/ParseException.cs

```diff
@@ -6,18 +6,18 @@ namespace Steeltoe.Common.Expression.Internal;
 
 public class ParseException : ExpressionException
 {
-    public ParseException(string expressionString, int position, string message)
-        : base(expressionString, position, message)
+    public ParseException(int position, string message)
+        : base(position, message)
     {
     }
 
-    public ParseException(int position, string message, Exception cause)
-        : base(position, message, cause)
+    public ParseException(string expressionString, int position, string message)
```

Comment:
I see the ordering changes in these classes, most of them before this one have seemed like clear readability improvements, but this one seems less orderly to me... How is the ordering being determined?

Reply:
It was a manual effort. `((int position, string message))` is the shortest, so should come first. But for the remaining two, I don't think one order is better over the other. This was a judgement call, but I'd be happy to swap them if you prefer.

Reply:
I don't think I have a strong preference, but I'd be inclined to sort by the first parameter. up to you

Reply:
I've swapped them.